OxfordBodleian/Juel-Jensen/BDLaethe33.xml

```diff
@@ -84,14 +90,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>A hole, presumably from parchment making, is found on <locus target="#40"/>.</p>
                                 </support>
                                 <extent>
                                     <measure unit="leaf">131</measure>
                                     <note>The foliation goes up to 130.
                                         Due to omission in numbering one leaf before <locus target="#28"/>, the two preceding leaves are numbered <locus target="#27a"/> and <locus target="#27b"/>.
```

Comment:
Due to an omission in the numbering of one leaf before... the two preceding leaves have been numbered as ...